pageserver/src/restore_local_repo.rs

```diff
@@ -26,7 +26,7 @@ use zenith_utils::lsn::Lsn;
 const MAX_MBR_BLKNO: u32 =
     pg_constants::MAX_MULTIXACT_ID / pg_constants::MULTIXACT_MEMBERS_PER_PAGE as u32;
 
-const ZERO_PAGE: Bytes = Bytes::from_static(&[0u8; 8192]);
+const ZERO_PAGE: &[u8; 8192] = &[0u8; 8192];
```

Comment:
I'm surprised by this change. It makes all the places that use ZERO_PAGE more verbose. What was the warning?

Reply:
Warning was about having const of type which is interior mutable, I've revisited it, initially I thought that replacing const with static would be enough, but I was concerned about possibility of mutation which is impossible because Bytes are immutable (there is a differrent type, BytesMut which can be mutable), so this changes to be just static, and ZERO_PAGE.clone() when using it. Also I found that we already use `static ZERO_PAGE: Bytes` probably we should create some zenith_consts to share that